### DISCUSSION TITLE
Push to a different repository

### DISCUSSION THREAD

**Discussion by domoritz** [community/community#26206] [Actions]
<p>I would like to push some commits to a different repo within the same org but the GH token doesn’t seem to be valid for other repos. What’s the best way to push code to other repos?</p>

**Comment by romaine-olympe** [comment]
<p>Hi,</p>
<p>I have same request do you have any solution about it ?</p>
<p>Awaiting your feedback about this <a href="https://fiduce.fr/services/programme-immobilier-neuf-ile-de-france/" title="case" rel="nofollow noopener">case</a>.</p>
<p>Thanks</p>
<hr>
<p>Romaine</p>

**Comment by jeremyepling** [comment]
<p>You need to create a <a href="https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line" rel="nofollow noopener">personal access token</a>.</p>

**Comment by domoritz** [comment]
<p>Thank you for the response. The PAT will have the same permissions as me, though, right? I’m a bit scared to give everyone in the repo access to all repos I have access to. Is there a way to scope the token? </p>

**Comment by jaredly** [comment, marked as answer]
<p>I haven’t found a way to scope the token, unfortunately – one workaround is to create a bot account and only add it to a couple of repos, and then make the token with that account.</p>

**Comment by ankit85jain** [comment]
<p>can you please share the solution u have implemented. i have to implement the same solution.</p>
